src/types.rs

```diff
@@ -456,6 +506,14 @@ pub enum Identifier {
     Dns(String),
 }
 
+impl fmt::Display for Identifier {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Self::Dns(domain) => write!(f, "{domain}"),
```

Comment:
My gut instinct here was that the identifier's `String` domain name is sufficient display and that adding a "DNS: " prefix isn't helpful. WDYT?

Reply:
Seems okay to me.